Author: Diane Trout <diane@ghic.org>
Description: Force libkpeople to conform to the convention that the library real name
 starts with the soname.

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -37,7 +37,7 @@
                        ${KDEPIMLIBS_KABC_LIBS}
 )
 
-set_target_properties (kpeople PROPERTIES VERSION ${KPEOPLE_VERSION} SOVERSION ${KPEOPLE_SOVERSION})
+set_target_properties (kpeople PROPERTIES VERSION ${KPEOPLE_SOVERSION}.${KPEOPLE_VERSION} SOVERSION ${KPEOPLE_SOVERSION})
 
 if (MSVC)
     set_target_properties (kpeople PROPERTIES
--- a/src/widgets/CMakeLists.txt
+++ b/src/widgets/CMakeLists.txt
@@ -28,7 +28,7 @@
                        kpeople
 )
 
-set_target_properties (kpeoplewidgets PROPERTIES VERSION ${KPEOPLE_VERSION} SOVERSION ${KPEOPLE_SOVERSION})
+set_target_properties (kpeoplewidgets PROPERTIES VERSION ${KPEOPLE_SOVERSION}.${KPEOPLE_VERSION} SOVERSION ${KPEOPLE_SOVERSION})
 
 if (MSVC)
     set_target_properties (kpeoplewidgets PROPERTIES
